my-app/src/app/components/section/section.component.ts

```diff
@@ -1,13 +1,18 @@
-import {Component} from '@angular/core';
+import {Component, OnInit} from '@angular/core';
 
 @Component({
   selector: 'app-section',
   templateUrl: './section.component.html',
   styleUrls: ['./section.component.scss']
 })
-export class SectionComponent {
+export class SectionComponent implements OnInit {
+  public courseName: string | undefined;
 
-  constructor() {
+  ngOnInit(): void {
+    this.courseName = ''
```

Comment:
The default value can be assigned to field at line 9: `public courseName: string = ''`

Reply:
Yes, fixed, thank you